### PR TITLE
Content tweaks

### PR DESCRIPTION
**Pull request by mattbennett**
Content tweaks, community section

### CODE REVIEW

**Comment by mattbennett** [comment]
This was working before the merge but now I get an obtuse `TypeError: Cannot read property 'site' of undefined`. Pretty sure it's a simple error but have tried and failed to debug it successfully :(

File: src/components/HomePage/Community.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  const boxStyles = css`
+    text-align: center;
+
+    ${media.desktop`
+      margin-bottom: 56px;
+    `};
+  `;
+
+  const linkStyles = css`
+  `;
```
Comment (kylesome) [inline src/components/HomePage/Community.tsx]: can remove this variable as there are no styles in it

File: src/components/HomePage/Community.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+  const imageStyles = css`
+  `;
```
Comment (kylesome) [inline src/components/HomePage/Community.tsx]: same here

File: src/components/HomePage/Community.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+  const imageWrapperStyles = css`
+  `;
```
Comment (kylesome) [inline src/components/HomePage/Community.tsx]: also here

**Review by kylesome — COMMENTED** [review]
Nice one on writing a new section! I updated some dependancies upstream so just run a `yarn install` again and it should work.

**Pull request merged**